docs/20.07-attributes.md

```diff
@@ -10,103 +10,29 @@ parent: Usage
 #### Component Attributes
 
 Attributes declared on the components will be all be accessible through the `state`.
-If the property is initialized in the `this.state`, the attribute will be reactive:
-
-```html
-<x-user status="thinking 🤔"><x-user>
-```
-
-`status` will therefore be reactive and the _thinking 🤔_ attribute value will overwrite the _Happy 😄_ default status.
-
-⚠️ A property that is not declared in the `state` won't be reactive.
-
-These properties can be accessed through `this.getAttribute()` from within the component.
-After all, these components are just native! 🏡
-
-
-#### Slots
-
-[Slots](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/slot) are part of the
-native web-component.
-Because Lego builds native web-components, you can use the standard _slots_ as documented.
+If the property is initialized in the `this.state`, the attribute will be reactive.
 
 Example:
 
 __index.html__
 ```html
-<user-profile>
-  <span>This user is in Paris</span>
-<user-profile>
-```
-
-__bricks/user-profile.html__
-```html
-<template>
-  <h1>User profile</h1>
-  <p>important information: <slot></slot></p>
-</template>
+<x-user status="Thinking 🤔"></x-user>

```

Comment:
Here is the example I have added and I just updated the script element to the new version.